movie-blog/src/components/create-element.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
You may go, like:
```
import React, { Component, createElement} from 'react'
```
To further use those as:
```
class CreateElement extends Component {
   ..
  const element = createElement(..
```
While above won't make your code much DRYer (as there would normally be just one component per file, hence one `Component` and `.createElement` invocation), I'd say it would be helpful to make yourself accustomed to that kind of syntax, which you may benefit, say, for function components (when there may be multiple `useState()` or `useEffect()` invocations).